sci-libs/pagmo/pagmo-9999.ebuild

```diff
@@ -9,9 +9,9 @@ PYTHON_COMPAT=( python2_7 )
 inherit cmake-utils git-r3 python-single-r1
 
 DESCRIPTION="Parallelization engine for optimization problems"
-HOMEPAGE="http://pagmo.sourceforge.net/"
+HOMEPAGE="https://github.com/esa/pagmo"
 SRC_URI=""
-EGIT_REPO_URI="git://pagmo.git.sourceforge.net/gitroot/pagmo/pagmo"
+EGIT_REPO_URI="https://github.com/esa/${PN}.git git://git@github.com:esa/${PN}.git"
```

Comment:
Okay, fixed that with commit https://github.com/gentoo-science/sci/commit/0e4c2d82cb4b6bfdc17daa4ec56246444e19c291. Actually `git://github.com/` is okay, but `git://git@github.com:` isn't. It was wrong in three of my ebuilds and unfortunately I referenced one of the broken ons. Sorry for the that, I probably should have noticed that during the pull request.